### PR TITLE
Zhanzhang

### PR DESCRIPTION
**Pull request by zhan-zhang**
Student Zhan Zhang added

### CODE REVIEW

**Review by amfleming — CHANGES_REQUESTED** [review]
Where is your status website?

**Comment by amfleming** [comment]
You need to resolve conflicts 

Looks like you have been working from old code - there are many students in the page now!

**Comment by amfleming** [comment]
This branch has conflicts that must be resolved

**Comment by zhan-zhang** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Lectures
 --------
 
 
+
```
Comment (amfleming) [inline README.md]: Why?

File: README.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ People
 
 #### Students
 <!-- Please keep in Alpha order -->
+
```
Comment (amfleming) [inline README.md]: Why?

File: README.md [hunk 3]
```diff
@@ -73,6 +75,7 @@ People
 * [Zen Yui](people/zen-yui.md) - jzy6 - [https://s3.amazonaws.com/jzy.io/grading.html](https://s3.amazonaws.com/jzy.io/grading.html)
 * [Rongxin Zhang](people/rongxin-zhang.md) - rz345 - [https://daqmewr4vl42v.cloudfront.net/grading.html](https://daqmewr4vl42v.cloudfront.net/grading.html)
 * [Yuxiang Zhu](people/yuxiang-zhu.md) - yz2329 - [https://s3.amazonaws.com/cs5356-yz2329/grading.html](https://s3.amazonaws.com/cs5356-yz2329/grading.html) 
+* [Zhan Zhang](people/zhan-zhang.md) - zz524 - [https://s3.us-east-2.amazonaws.com/zhanzhang/grading.html](https://s3.us-east-2.amazonaws.com/zhanzhang/grading.html)
```
Comment (amfleming) [inline README.md]: Please keep student names in alphabetical order

File: people/zhan-zhang.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Zhan Zhang
+------------
+
+![](photos/zhan-zhang.png)
+
```
Comment (amfleming) [inline people/zhan-zhang.md:5]: Great!

**Review by amfleming — CHANGES_REQUESTED** [review]

**Comment by amfleming** [comment]
Almost!

**Comment by zhan-zhang** [comment]
updated @amfleming

**Pull request merged**